doc/source/users/environment_variable_catalog.rst

```diff
@@ -0,0 +1,352 @@
+    changes (up to and including removal of this variable) may occur if
+    deemed necessary by the core developers.
+
+.. _directorymap:
+
+DirectoryMap
+~~~~~~~~~~~~
+*Not mandatory, set by user*
+
+In case that the server and the client do not run on the same host, the scan
+data may be easily shared between them using the NFS. Since some of the
+tools e.g. showscan rely on the scan data file the DirectoryMap may help in
+overcoming the shared directory naming issues between the hosts.
+
+Its value is a dictionary with keys pointing to the server side directory
+and values to the client side directory/ies (string or list of strings).
+
+.. todo::
+    Add an example here.
+
+.. _jsonrecorder:
+
+JsonRecorder
+~~~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Enable/disable the JsonRecorder. This recorder is used to retrieve the data
+generated by a macro during its run time in a json format.
+An example of usage of this recorder is plotting live scans.
+
+.. todo:: Add reference to the jsonrecorder documentation when available.
+
+.. _outputblock:
+
+OutputBlock
+~~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Enable/disable printing the scan output records at the same line instead
+of appending records as new lines a.k.a. block mode. This can save space
+of the console output and facilitate an easy visual access to the scan
+header printed at the beginning of the scan - especially useful for very
+long scans.
+
+.. _outputcols:
+
+OutputCols
+~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Select a subset of the scan output columns to be printed. Two different ways
+of specifying the selected columns exists. First, by specifying an integer
+number e.g. 2 means print just the very first two columns.
+Second, by specifying a list of string identifiers e.g. scan elements URIs
+or the scan record metadata identifiers: "point_nb", "timestamp".
+
+The original order of colums is not changed. This feature is similar to the
+*expconf* enable/disable output.
+
+Example:
+
+::
+
+    senv OutputCols "['tango_host:port/expchan/zerodctrl01/1','motor1','timestamp']"
+
+.. _prescansnapshot:
+
+PreScanSnapshot
+~~~~~~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Variable listing the elements that will be collected and saved before a Scan
+is executed.
+
+The format for the value is the following:
+
+::
+
+[(taurus model URI, label)]
+
+If the first element in the tuple is a Sardana element, it's enough to set up
+the URI up to the tango device server name. If the element is outside Sardana,
+i.e. A Tango attribute, you need to add the full URI, up to the attribute name.
+
+Exemple:
+
+::
+
+[('tango://sardana/motor/01', 'mot01')]
+
+or
+
+::
+
+[('tango://device/server/01/attribute','label')]
+
+.. note:: You can use :ref:`expconf <expconf_ui>` to set up pre-scan snapshot.
+
+.. _sampleinfo:
+
+SampleInfo
+~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Extra info for the sample. describe la muestra.
```

Comment:
Should be better explained, possibly not in Spanish ;)

Reply:
Uuuupppsss ... that shouldn't be in this PR. I'm going to fix it.